src-juce/AWConsolidatedEditor.cpp

```diff
@@ -2080,6 +2242,45 @@ juce::PopupMenu AWConsolidatedAudioProcessorEditor::makeSettingsMenu(bool withHe
         getSettingsDirectory(false).getChildFile("customDocs").revealToUser();
     });
 
+    settingsMenu.addSeparator();
+    juce::PopupMenu layout;
+    juce::String busLayout{"Unknown -> "};
+    if ( processor.getTotalNumInputChannels() == 1 )
+        busLayout = "Mono -> ";
+    else if (processor.getTotalNumInputChannels() == 2 )
+        busLayout = "Stereo -> ";
+    if ( processor.getTotalNumOutputChannels() == 1 )
+        busLayout += "Mono";
+    else if (processor.getTotalNumOutputChannels() == 2 )
+        busLayout += "Stereo";
+    else
+        busLayout += "Unknown";
+
+    layout.addSectionHeader("Mono behaviour");
+    layout.addSeparator();
+    layout.addItem("Allow stereo plugins in mono mode (global)", true, processor.properties->getBoolValue("stereoPluginsInMono", true),
+        [w = juce::Component::SafePointer(this)]() {
+            if (!w)
+                return;
+            w->processor.properties->setValue("stereoPluginsInMono", !w->processor.properties->getBoolValue("stereoPluginsInMono", true));
+        });
+    layout.addSeparator();
+    layout.addItem("Output only L channel", true, *processor.monoBehaviourParameter == AWConsolidatedAudioProcessor::MonoBehaviourParameter::LeftOnly,
+        [w = juce::Component::SafePointer(this)]() {
+            if (!w)
+                return;
+            *w->processor.monoBehaviourParameter = AWConsolidatedAudioProcessor::MonoBehaviourParameter::LeftOnly;
+        });
+    layout.addItem("Output L+R channel", true, *processor.monoBehaviourParameter == AWConsolidatedAudioProcessor::MonoBehaviourParameter::LeftRightSum,
+        [w = juce::Component::SafePointer(this)]() {
+            if (!w)
+                return;
+            *w->processor.monoBehaviourParameter = AWConsolidatedAudioProcessor::MonoBehaviourParameter::LeftRightSum;
+        });
+    layout.addSeparator();
+    layout.addItem("Bus layout: " + busLayout, false, false, []() {});
+    settingsMenu.addSubMenu("Layout", layout);
```

Comment:
This is the "Layout" to "Channel Layout" I think we want to swap

Reply:
Yes, fixed